utils/consts.go

```diff
@@ -2382,6 +2381,18 @@ const (
 	OptsRouteID = "*routeID"
 	// EEs
 	OptsEEsVerbose = "*eesVerbose"
+	// EEs Elasticsearch options
+	Index               = "index"
```

Comment:
Please prefix the options with something like EEs so we know they belong to the same place.